src/controllers/accountsSettlementController.ts

```diff
@@ -2,13 +2,13 @@ import { Context } from 'koa'
 import { Redis } from 'ioredis'
 import { normalizeAsset } from '../utils/normalizeAsset'
 
-export async function create (ctx: Context) {
+export async function create(ctx: Context) {
   const accountJson = await ctx.redis.get(`${ctx.settlement_prefix}:accounts:${ctx.params.id}`)
   const account = JSON.parse(accountJson)
 
-  const body = ctx.request.body
-  const amount = normalizeAsset(body.scale, 6, BigInt(body.amount))
-  await ctx.settleAccount(account, amount.toString())
+  const amount = ctx.request.body.amount;
+  // the scale should be configured on the connector
+  await ctx.settleAccount(account, +amount)
```

Comment:
Why are you changing the amount to a number?

We use strings or bigint for amounts because we use UInt64 numbers which may be too large to safely parse as Javascript Numbers.